packages/fluentui/react-northstar/src/themes/teams/components/Checkbox/checkboxIndicatorIndeterminatedUrl.ts

```diff
@@ -0,0 +1,6 @@
+export default (color: string, backgroundColor: string) =>
+  `url("data:image/svg+xml, %3Csvg xmlns='http://www.w3.org/2000/svg' role='presentation' style='background-color: ${escape(
```

Comment:
Move this to the component where it is used

Reply:
Done